internal/provider/security/security_account_resource.go

```diff
@@ -468,7 +468,7 @@ func (r *SecurityAccountResource) Update(ctx context.Context, req resource.Updat
 	}
 
 	// locked update
-	if !plan.Locked.IsNull() {
+	if !plan.Locked.Equal(state.Locked) && !plan.Locked.IsNull() && !plan.Locked.IsUnknown() {
```

Comment:
with this change, then in the case that created an account with locked = true, then comment out locked = true on the resource file, then doing terraform apply again won't tell any change. However, locked default value is false. In the resource, locked parameter was not mentioned, so ideally people should treat it using default value which is false.
The state file and the real are showing locked is true. It does not aligned with the expectation.